ccinput/constants.py

```diff
@@ -1608,6 +1610,16 @@ class CalcType(Enum):
         "gfnff": "gfn-ff",
         "gfn2xtbgfnff": "gfn2-xtb//gfn-ff",
     },
+    "psi4": {
+        "scf": "scf",
```

Comment:
With the modification above, this line will not be necessary anymore, since the canonical method name will be `hf`.

Reply:
Done.